wire-desktop-beta/.SRCINFO

```diff
@@ -1,34 +1,24 @@
 pkgbase = wire-desktop-beta
-	pkgdesc = Modern, private messenger. Based on Electron.
+	pkgdesc = End-to-end encrypted messenger with file sharing, voice calls and video conferences
 	pkgver = 3.5.2881
-	pkgrel = 1
+	pkgrel = 2
 	url = https://wire.com/
 	arch = x86_64
-	arch = i686
 	license = GPL3
 	makedepends = cargo
 	makedepends = npm
 	makedepends = python2
 	makedepends = git
 	makedepends = yarn
-	depends = alsa-lib
-	depends = gconf
-	depends = gtk2
-	depends = libxss
-	depends = libxtst
-	depends = nss
+	depends = electron
 	depends = xdg-utils
-	optdepends = hunspell-en: for English spellcheck support
-	optdepends = noto-fonts-emoji: for colorful emoji made by Google
-	optdepends = ttf-emojione: for colorful emoji made by EmojiOne
+	optdepends = hunspell-en_US: for English spellcheck support
+	optdepends = emoji-font: colorful emoji
```

Comment:
just noticed that AUR interface is helpful here too, check this out:

![image](https://user-images.githubusercontent.com/1177900/50538655-c02f1880-0b72-11e9-917e-05a5347216d2.png)